src/include/function/table_functions/bind_input.h

```diff
@@ -8,8 +8,8 @@ namespace kuzu {
 namespace function {
 
 struct TableFuncBindInput {
-    explicit TableFuncBindInput(std::vector<common::Value> inputs) : inputs{std::move(inputs)} {}
-    std::vector<common::Value> inputs;
+    explicit TableFuncBindInput(std::vector<common::Value*> inputs) : inputs{std::move(inputs)} {}
+    std::vector<common::Value*> inputs;
```

Comment:
I doubt this is safe. Do we have guarantee of the life cycle of these pointers? If not better to copy.